mathesar_ui/src/sections/table-view/Header.scss

```diff
@@ -0,0 +1,20 @@
+.add-column {
+    padding:0.5em;
+    overflow:hidden;
+
+    .grid {
+        display:grid;
+        grid-template-columns:auto auto;
+        grid-gap: 0 0.5em;
+    }
+
+    .messages {
+        color:darkred;
+        font-style: italic;
+
+        strong {
+            font-weight:bold;
+            font-style:normal;
+        }
+    }
+}
```

Comment:
Please add a newline at the end of all files. This is a convention we follow.

Feel free to raise an issue to add a eslint rule to force this.

Reply:
I have checked in the file with newline as specified and have updated my IDE settings.